static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -37,7 +37,7 @@ const PageFilterBar = styled('div')<{condensed?: boolean}>`
   }
 
   & > * {
-    min-width: 6rem;
+    min-width: 7rem;
```

Comment:
Increasing this to 7rem ensures that `24h` and other 3 character time range selections don't overflow